### PR TITLE
Fix broken validation tips and typo in ubuntu-multi-nodes-guide

### PR DESCRIPTION
**Pull request by resouer**
Fix:
1. validation tips broke the list in doc
2. fix typo in command line

### CODE REVIEW

**Comment by zmerlynn** [comment]
LGTM

File: docs/getting-started-guides/ubuntu_multinodes_cluster.md [hunk 1]
```diff
@@ -112,15 +114,14 @@ If you want a node acts as **both running the master and minion**, please choose
 	
 	> `$ /opt/bin/etcdctl get /coreos.com/network/config`
 	
-	> If you got `{"Network":"10.0.0.0/16"}`，then etcd cluster is working in good condition. **Victory is in sight！**
-	> If not , you should check` /var/log/upstart/etcd.log` to resolve etcd problem before going forward.
+	> If got `{"Network":"10.0.0.0/16"}`, then etcd cluster is working well.
```
Comment (rjnagal) [inline docs/getting-started-guides/ubuntu_multinodes_cluster.md]: s/If got/If you got/

**Comment by bgrant0607** [comment]
Please fix the typo, and we'd be happy to merge.

**Comment by resouer** [comment]
@rjnagal  Aha, I'll fix it soon.  btw, do you think it is reasonable to change the file name to `ubuntu_multi_nodes_cluster.md`? I'm also not sure if its current name is linguistic right ...

**Comment by rjnagal** [comment]
I think we should just merge the multiple ubuntu guides into one ubuntu.md.
Running everything on a single node should just be a special case of
multi-node configuration.

**Comment by resouer** [comment]
@bgrant0607 @rjnagal  Typo fixed. We are also informed that single node setup should be deprecated soon. We'd like to give that a shoot if

**Comment by bgrant0607** [comment]
LGTM for this PR. Merging multi-node and single-node can be a separate PR.

**Pull request merged**